testing/deployer/sprawl/boot.go

```diff
@@ -372,6 +375,11 @@ func (s *Sprawl) preRegenTasks() error {
 		return fmt.Errorf("registerAllServicesForDataplaneInstances: %w", err)
```

Comment:
I wonder if it would be clearer to do the register/deregister jointly in one iteration rather than serially like this in two separate _very similar_ iterations.

Perhaps the helper methods would then need to be renamed as something like `syncXXX` instead of `register/deregisterXXX`

Reply:
Sounds good. I will combine these two methods.